web/components/VideoWithMetadata.vue

```diff
@@ -40,6 +50,7 @@ const props = defineProps<{
 
 const c2paValidationError = ref('')
 const c2paManifestStore = ref<ManifestStore | null>(null)
+const isLoadingC2PAManifestStore = ref(true)
```

Comment:
We should set a loading flag for `useFetch` in `readURLAsBlob` since that's where we download the whole video file and needs to wait for a load time. This together with c2pa loading should be a computed flag we use in UI logic

Reply:
Updated